src/VisualStudio/IntegrationTest/TestUtilities/InProcess/FindReferencesWindow_InProc.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Linq;
+using EnvDTE80;
+using Microsoft.VisualStudio.IntegrationTest.Utilities.Common;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Shell.TableControl;
+using Microsoft.VisualStudio.Shell.TableManager;
+
+namespace Microsoft.VisualStudio.IntegrationTest.Utilities.InProcess
+{
+    internal class FindReferencesWindow_InProc : InProcComponent
+    {
+        public static FindReferencesWindow_InProc Create() => new FindReferencesWindow_InProc();
+
+        public Reference[] GetContents(string windowCaption)
+        {
+            return InvokeOnUIThread(() =>
+            {
+                // Find the tool window
+                var toolWindow = ((DTE2)GetDTE()).ToolWindows.GetToolWindow(windowCaption);
+
+                // Dig through to get the Find References control.
+                var toolWindowType = toolWindow.GetType();
+                var toolWindowControlField = toolWindowType.GetField("Control");
+                var toolWindowControl = toolWindowControlField.GetValue(toolWindow);
```

Comment:
@KirillOsenkov always just made an extension method to do the GetType/GetField/GetValue dance, which might be nice to have here.

Do we have a bug tracking getting something better for this?